nbs/tsdataset.ipynb

```diff
@@ -54,11 +54,12 @@
     "#| export\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #82.            last_times = np.array([], dtype=np.datetime64)
</pre><p>Please make this a regular list. We also support having integer times, so I think it's better to use a regular list, append all the last times and then call np.array on that (should also be faster as well)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/neuralforecast/pull/1049/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
Makes sense - I've even just passed that list directly into the pd.Index() structure, so avoided the explicit conversion to numpy